packages-web/datagrid/src/Datagrid.editorPreview.tsx

```diff
@@ -0,0 +1,43 @@
+import { createElement, ReactElement } from "react";
+import { DatagridPreviewProps } from "../typings/DatagridProps";
+
+import "./ui/Datagrid.scss";
+import { Table } from "./components/Table";
+
+export function preview(props: DatagridPreviewProps): ReactElement {
+    const data = Array.from({ length: props.pageSize ?? 5 }).map(() => ({
+        id: "" as any
+    }));
+
+    return (
+        <Table
+            className={props.class}
+            columns={props.columns}
+            data={data}
+            hasMoreItems={false}
+            page={0}
+            columnsDraggable={props.columnsDraggable}
+            columnsFilterable={props.columnsFilterable}
+            columnsResizable={props.columnsResizable}
+            columnsSortable={props.columnsSortable}
+            columnsHidable={props.columnsHidable}
+            pageSize={props.pageSize ?? 5}
+            paging={props.pagingEnabled}
+            pagingPosition={props.pagingPosition}
+            footerWidgets={
+                <props.footerWidgets.renderer>
```

Comment:
🎄  This makes me think that maybe it would be nice to try handle differences between preview and actual cells also within this file... Otherwise our trick with `id: "" as any` looks awkward

Reply:
Not sure if I understand

Reply:
This is related to `{/* TODO: Not just a div */}` we forgot in `Table` :) Basically, my point is that `Table` handles the difference between preview and production only for `column.content` but not for footer/header. This looks misaligned: either do all or none. I vote for none, but not sure how simple it is from code organization perspective.

Reply:
Dont know the final approach.. they are different because 1 we use inside the cell as renderer and the other ones are just provided by the platform and rendered as react elements..

Reply:
As we talked we came up with a better solution :)